src/components/header/menu/headerMenu.component.tsx

```diff
@@ -157,14 +160,15 @@ const ItemWrapper = styled.div`
   }
 `;
 
-const Item = styled.button`
+const Item = styled.button<{ dark: boolean }>`
```

Comment:
```suggestion
const Item = styled.button<{ dark?: boolean }>`
```
아래 코드 너무 많이 바뀌는 거 옵셔널로 해결해주세요

Reply:
옵셔널로 바꾼다 해서 코드 많이 바뀌는게 해결이 되는지 모르겠어요.

다른 방법으로  Item 컴포넌트마다 dark props 전달하는거 고쳤어요